tools/vsimporter/xib2nib/UIControl.cpp

```diff
@@ -35,6 +36,39 @@ void UIControl::InitFromXIB(XIBObject *obj)
 void UIControl::InitFromStory(XIBObject *obj)
 {
     UIView::InitFromStory(obj);
+
+    const char* horizontalAlign = getAttrib("contentHorizontalAlignment");
+    if (horizontalAlign) {
+        if (strcmp(horizontalAlign, "center") == 0) {
+            _contentHorizontalAlignment = (int)UIControlContentHorizontalAlignmentCenter;
+        } else if (strcmp(horizontalAlign, "left") == 0) {
+            _contentHorizontalAlignment = (int)UIControlContentHorizontalAlignmentLeft;
+        } else if (strcmp(horizontalAlign, "right") == 0) {
+            _contentHorizontalAlignment = (int)UIControlContentHorizontalAlignmentRight;
+        } else if (strcmp(horizontalAlign, "fill") == 0) {
+            _contentHorizontalAlignment = (int)UIControlContentHorizontalAlignmentFill;
+        } else {
+            assert(0);
+        }
+        getAttrAndHandle("contentHorizontalAlignment");
+    }
+
+    const char* verticalAlign = getAttrib("contentVerticalAlignment");
+    if (verticalAlign) {
+        if (strcmp(verticalAlign, "center") == 0) {
+            _contentVerticalAlignment = (int)UIControlContentVerticalAlignmentCenter;
+        } else if (strcmp(verticalAlign, "top") == 0) {
+            _contentVerticalAlignment = (int)UIControlContentVerticalAlignmentTop;
+        } else if (strcmp(verticalAlign, "bottom") == 0) {
+            _contentVerticalAlignment = (int)UIControlContentVerticalAlignmentBottom;
+        } else if (strcmp(verticalAlign, "fill") == 0) {
+            _contentVerticalAlignment = (int)UIControlContentVerticalAlignmentFill;
+        } else {
+            assert(0);
```

Comment:
We typically don't assert even though we might have a few legacy places where this is still done. Instead, just replace it with printf("Unknown verticalightment value: %s\n", verticalAlign) for now